_posts/2017-08-08-CompetitiveCoding.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title: "Competitive Coding: From Novice to Expert"
```

Comment:
I am not talking about a particular topic in the post (like posts by others) that you can judge it as "novice" or "expert"...the post just brings out differences between a novice and an expert competitive coder.